x-pack/plugins/uptime/server/lib/alerts/status_check.ts

```diff
@@ -291,7 +301,9 @@ export const statusCheckAlertFactory: UptimeAlertTypeFactory = (_server, libs) =
         for (const monitorLoc of downMonitorsByLocation) {
           const monitorInfo = monitorLoc.monitorInfo;
 
-          const alertInstance = alertInstanceFactory(MONITOR_STATUS.id + monitorLoc.location);
+          const urlText = monitorInfo.url?.full?.replace(/[^A-Z0-9]+/gi, '_').toLowerCase();
```

Comment:
This regexp is repeated below. Can we extract it to a helper function `normalizeURL`, for instance?

Reply:
done in [`eb400a9` (#81736)](https://github.com/elastic/kibana/pull/81736/commits/eb400a9d1dfad99ff521dd9847b5b63ddeeaa48d)